src/blacklist/5_0_0/sysd/blacklist_sysd.tex

```diff
@@ -0,0 +1,202 @@
+  \label{sec:model:#2} \label{sec:model:#2s} \label{sec:model:#2es}
+}
+
+\begin{document}
+
+%% Arrowhead Document Properties
+\ArrowheadTitle{Blacklist Support System}
+\ArrowheadType{System Description}
+\ArrowheadTypeShort{SysD}
+\ArrowheadVersion{5.0.0}
+\ArrowheadDate{\today}
+\ArrowheadAuthor{Katinka Jakó}
+\ArrowheadStatus{DRAFT}
+\ArrowheadContact{jako.katinka@aitia.ai}
+\ArrowheadFooter{\href{www.arrowhead.eu}{www.arrowhead.eu}}
+\ArrowheadSetup
+%%
+
+%% Front Page
+\begin{center}
+  \vspace*{1cm}
+  \huge{\arrowtitle}
+
+  \vspace*{0.2cm}
+  \LARGE{\arrowtype}
+  \vspace*{1cm}
+
+  %\Large{Service ID: \textit{"\arrowid"}}
+  \vspace*{\fill}
+
+  % Front Page Image
+  %\includegraphics{figures/TODO}
+
+  \vspace*{1cm}
+  \vspace*{\fill}
+
+  % Front Page Abstract
+  \begin{abstract}
+    This document provides system description for the \textbf{Blacklist Support System}.
+  \end{abstract}
+
+  \vspace*{1cm}
+
+ \end{center}
+
+\newpage
+%%
+
+%% Table of Contents
+\tableofcontents
+\newpage
+%%
+
+\section{Overview}
+\label{sec:overview}
+\color{black}
+This document describes the Blacklist Support system, which makes it possible to ban systems from the Eclipse Arrowhead Local Cloud (LC), so that their requests will not be served. This system provides storage functionality for the blacklist entries that contain information about the ban for each system.
+
+The rest of this document is organized as follows.
+In Section \ref{sec:prior_art}, we reference major prior art capabilities
+of the system.
+In Section \ref{sec:use}, we describe the intended usage of the system.
+In Section \ref{sec:properties}, we describe fundamental properties
+provided by the system.
+In Section \ref{sec:delimitations}, we describe delimitations of capabilities
+of the system.
+In Section \ref{sec:services}, we describe the abstract services produced by the system.
+In Section \ref{sec:security}, we describe the security capabilities
+of the system.
+
+\subsection{Significant Prior Art}
+\label{sec:prior_art}
+
+The strong development on cloud technology and various requirements for digitization and automation has led to the concept of Local Clouds (LC).
+
+\textit{"The concept takes the view that specific geographically local automation tasks should be encapsulated and protected."} \cite{jerker2017localclouds}
+
+One of the main building blocks when realizing such Local Cloud is the capability of centrally and consistently control the blacklist within the given LC, and store who was banned, when and why. The Blacklist Support system was not part of the LC in the previous versions (4.6.x), but it is a recommended Support system in versions 5.0.0 and higher.
+
+
+\subsection{How This System Is Meant to Be Used}
+\label{sec:use}
+
+Blacklist is a recommended Support system of the Eclipse Arrowhead Local Cloud and is responsible to provide a central blacklist functionality by storing the blacklist entries.
```

Comment:
need a second sentence as an example. Maybe something about that an operator ban a consumer because it is flooding  the LC with thousands of requests.